package.nls.ja.json

```diff
@@ -0,0 +1,97 @@
+{
+	"d.switchConfiguration": "code-d: 設定の切替",
+	"d.action.switchConfiguration": "設定の切替",
+	"d.switchArchType": "code-d: アーキテクチャタイプの切替",
+	"d.action.switchArchType": "アーキテクチャタイプの切替",
+	"d.switchBuildType": "code-d: ビルドタイプの切替",
+	"d.action.switchBuildType": "ビルドタイプの切替",
+	"d.switchCompiler": "code-d: コンパイラの切替",
+	"d.action.switchCompiler": "コンパイラの切替",
+	"d.killServer": "code-d: DCD Server の強制終了",
+	"d.action.killServer": "DCD Server の強制終了",
+	"d.restartServer": "code-d: DCD Server の再起動",
+	"d.action.restartServer": "DCD Server の再起動",
+	"d.reloadImports": "code-d: インポートパスの再読み込み",
+	"d.action.reloadImports": "インポートパスの再読み込み",
+	"d.run": "code-d: プロジェクトを実行",
+	"d.action.run": "プロジェクトを実行",
+	"d.build": "code-d: プロジェクトをビルド",
+	"d.action.build": "プロジェクトをビルド",
+	"d.stop": "code-d: ビルドを停止",
+	"d.action.stop": "ビルドを停止",
+	"d.debug": "code-d: code-debug を使用してプロジェクトをデバッグ",
+	"d.action.debug": "code-debug を使用してプロジェクトをデバッグ",
+	"d.uploadSelection": "code-d: エディタで選択した部分を dpaste.com にアップロード",
+	"d.action.uploadSelection": "エディタで選択した部分を dpaste.com にアップロード",
+	"d.insertDscanner": "code-d: デフォルトの dscanner.ini を挿入する",
+	"d.action.insertDscanner": "デフォルトの dscanner.ini を挿入する",
+	"d.rdmdCurrent": "rdmd で実行",
+	"d.generateCoverageReport": "code-d: コードカバレッジレポートを生成",
+	"d.action.generateCoverageReport": "コードカバレッジレポートを生成",
+	"d.showGCCalls": "code-d: GCコールを表示する(profilegc.log)",
+	"d.action.showGCCalls": "GCコールを表示する(pofilegc.log)",
+	"dub.openSettingsEditor": "code-d/dub: プロジェクト設定を開く",
+	"dub.action.openSettingsEditor": "プロジェクト設定を開く",
+	"dub.closeSettingsEditor": "code-d/dub: プロジェクト設定を閉じる",
+	"dub.action.closeSettingsEditor": "プロジェクト設定を閉じる",
+	"d.createProject": "code-d: 新しいプロジェクトを作成",
+	"d.action.createProject": "新しいプロジェクトを作成",
+
+	"d.config.workspacedPath": "workspace-d のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.stdlibPath": "標準ライブラリ(Phobos,D Runtime)のフルパスの配列を設定します、指定しない場合はオートコンプレートは動作できません",
+	"d.config.dcdClientPath": "dcd-client のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.dcdServerPath": "dcd-server のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.dscannerPath": "dscanner のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.dfmtPath": "dfmt のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.dubPath": "dub のファイルのパスを設定します 環境変数PATHに設定されている、または code-d でインストールした場合は省略できます",
+	"d.config.enableLinting": "構文チェック(Linting)を有効にします",
+	"d.config.enableSDLLinting": "dub.sdl ファイルの構文チェック(Linting)を有効にします",
+	"d.config.enableDubLinting": "dub が行う構文チェック(Linting)を有効にします",
+	"d.config.enableAutoComplete": "オートコンプレートの dcd-server/dcd-client をスタートアップします",
+
```

Comment:
Can you fix the empty lines here so they are the same as in the package.nls.json file?